package.json

```
@@ -11,6 +11,7 @@
     "url": "https://github.com/mocha-community/json-file-reporter.git"
   },
   "scripts": {
+    "lint": "eslint .",
```

Please use `--ignore-path` option.

```suggestion
    "lint": "eslint --ignore-path .gitignore .",
```

cf. https://eslint.org/docs/user-guide/configuring#using-an-alternate-file

---

ESLint 7 ignores `node_modules` by default. Still want the change?